pkg/macos/build.sh

```diff
@@ -189,7 +189,7 @@ printf "v%.0s" {1..80}; printf "\n"
 #-------------------------------------------------------------------------------
 # Build Python
 #-------------------------------------------------------------------------------
-"$SCRIPT_DIR/build_python.sh"
+"$SCRIPT_DIR/build_python.sh" -v 3.10.12 -r 0.13.4
```

Comment:
```suggestion
"$SCRIPT_DIR/build_python.sh" -v 3.10.12 -r 0.13.4
```
Why are we passing these versions directly in this script?
I'd prefer the `build.sh` to accept argument for these versions and pass that along when calling `build_python.sh`...

Or, grab these versions from `cicd/shared-gh-workflows-context.yml` ...

Reply:
I can add those

Reply:
just wondering if this should be Python 3.10.13 since that is the latest ?

Reply:
> just wondering if this should be Python 3.10.13 since that is the latest ?

No, we still don't have the relenv release for that, although, when building from source it could be that version.

But that's exactly the problem.
We need to define which python version we should be using, **in 1 place**, then, read it and pass that along...

Reply:
The script now gets the Relenv and Python versions from `cicd/shared-gh-workflows-context.yml` if no version is passed